src/core_modules/capture-core/components/Pages/ViewEvent/EventDetailsSection/EditEventDataEntry/editEventDataEntry.epics.js

```diff
@@ -2,6 +2,7 @@
 import { ActionsObservable } from 'redux-observable';
 import { batchActions } from 'redux-batched-actions';
 import { moment } from 'capture-core-utils/moment';
```

Comment:
looks like this is not used here as well. Since we touched this file in this PR can we pretty please delete it?

Reply:
Caused by adding the default for the momentDate parameter in getFormattedStringFromMomentUsingEuropeanGlyphs function. I will keep the parameter without default so still need this here.